### ISSUE TITLE
Switch instance and switch to orginal 

### ISSUE THREAD

**Issue by Jpxe** [benbusby/farside#123]
It would be great with a way to switch instance and a way to switch to the original service. 

Maybe it could be done with Redirector, I posted a feature request on their GitHub: https://github.com/einaregilsson/Redirector/issues/362. 

Or is there some way to do this that I'm not aware of?

**Comment by pokegamer5547** [comment]
You can use _/ to switch instances.  For example, when you open https://farside.link/_/nitter and then want to switch instance, you just have to press back once and farside will use another instance.

**Comment by Jpxe** [comment]
@pokegamer5547 can I use that in an extension?

**Comment by pokegamer5547** [comment]
> @pokegamer5547 can I use that in an extension?

@Jpxe i never used it with extensions since i only have android device as of now. But i dont see why you can't use it.

**Comment by Jpxe** [comment]
@pokegamer5547 I don't think your suggestion works in desktop, or maybe I'm misunderstanding something? If I go to that link you provided in a new tab it redirect to an instance and my back button is grayed out. If I open it in this tab and press back it goes back to this page.

**Comment by pokegamer5547** [comment]
> 

@Jpxe Strange. Is the redirect page showing to you? Like, the page which says "redirecting to nitter". If so, i can't really help much since im not on desktop :(

**Comment by Jpxe** [comment]
@pokegamer5547 yeah it shows but it doesn't seem to be in the back button history so back button skips it

**Comment by pokegamer5547** [comment]
> @pokegamer5547 yeah it shows but it doesn't seem to be in the back button history so back button skips it

@Jpxe is that page "redirecting" page showing in your browser history though?

**Comment by Jpxe** [comment]
@pokegamer5547 yes

**Comment by pokegamer5547** [comment]
> @pokegamer5547 yes

@Jpxe please try it in different browser and see if works or not.
If not, then i suppose there's nothing i can help you with :/

**Comment by Jpxe** [comment]
@pokegamer5547 ah it worked in Firefox. It doesn't work in Brave and that's the browser I use normally at the moment. I don't think I'll switch browsers just for this but it's good to know. Thanks for your help!

**Comment by Bakr-Ali** [comment]
> If I go to that link you provided in a new tab it redirects to an instance and my back button is grayed out. If I open it in this tab and press back it goes back to this page.

I'm having the same behavior on Edge and Opera too.

And to add to this part: "If I open it in this tab and press back it goes back to this page." Long pressing/clicking on the back button in this case shows the "redirecting" page, and clicking it actually does go back to it and redirects to another instance!
